docs/ReduxIntegration.md

```diff
@@ -0,0 +1,377 @@
+---
+id: redux-integration
+title: Redux Integration
+---
+
+This section deals with testing RN applications developed with Redux. We will be developing a simple TODO application capable of adding and removing an item. Once included, the timestamp is included.
+
+## Setting up
+
+Install the packages required for Redux and it up. For this example we will simply install `react-redux` and `redux`. We will then proceed to create our action creators (found in `./actions/todoActions.js`) and our reducers (found in `./reducers/index.js`)
```

Comment:
Avoid using "simply", "just", etc. Stuff is not simple or obvious for everybody and it may discourage some :) 
```suggestion
Install the packages required for Redux and it up. For this example we will install `react-redux` and `redux`. We will then proceed to create our action creators (found in `./actions/todoActions.js`) and our reducers (found in `./reducers/index.js`)
```